src/wp-includes/compat.php

```diff
@@ -614,7 +614,7 @@ function array_last( array $array ) { // phpcs:ignore Universal.NamingConvention
 	define( 'IMG_AVIF', IMAGETYPE_AVIF );
 }
 
-// IMAGETYPE_HEIC constant is not yet defined in PHP as of PHP 8.3.
+// IMAGETYPE_HEIC constant is only defined in PHP 8.5 or later.
 if ( ! defined( 'IMAGETYPE_HEIC' ) ) {
-	define( 'IMAGETYPE_HEIC', 99 );
+	define( 'IMAGETYPE_HEIC', 20 );
```

Comment:
Where is the `20` coming from? And where was `99` coming from before?

Reply:
I don't see it in the linked commit: https://github.com/php/php-src/commit/dfac2da7fba930570fcda99205cfcf53dbe2e909

Reply:
I believe this was just a placeholder since there was no set value.